refinery/factory_boy/utils.py

```diff
@@ -66,17 +70,36 @@ def create_dataset_with_necessary_models():
     investigation_uuid = str(uuid_builtin.uuid4())
     investigation = InvestigationFactory(uuid=investigation_uuid)
 
-    study_uuid = str(uuid_builtin.uuid4())
-    StudyFactory(
-        uuid=study_uuid,
-        investigation=investigation,
-        description="This is a great DataSet"
-    )
-
     InvestigationLinkFactory(
         data_set=dataset,
         investigation=investigation,
         version=1,
         date=datetime.now()
     )
+
+    study_uuid = str(uuid_builtin.uuid4())
+    study = StudyFactory(
+        uuid=study_uuid,
+        investigation=investigation,
+        description="This is a great DataSet"
+    )
+
+    for i in xrange(2):
+        assay_uuid = str(uuid_builtin.uuid4())
+        assay = AssayFactory(
+            uuid=assay_uuid,
+            study=study
+        )
+        attribute = AttributeFactory()
```

Comment:
@scottx611x : huh, but right below it looks like like NodeFactory needs an attribute? This is the chicken-and-egg problem I was stuck on.